cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/ExecutableBuilder.java

```diff
@@ -332,6 +341,15 @@ public ExecutableModellingResult transformToExecutable(ParsedSlang parsedSlang,
         }
     }
 
+    private void validateOutputs(List<Output> outputs) {
+        for (Output output: outputs) {
+            if (output.hasRobotProperty()) {
+                throw new RuntimeException("'robot' property allowed only for outputs of sequential_action. " +
```

Comment:
This throws the error at the first invalid output. I would go with with something like:
return outputs.stream().filter(r->r.has..()).map(r->new RTE()).collect(toList()) and all to the errors list.